content/maintainers-guide/how-to-become-a-committer.md

```diff
@@ -0,0 +1,61 @@
+---
+title: How to become a committer
+weight: 12
+---
+
+# How to become a committer
+
+In the PrestaShop project, committer are contributors with partial merge rights. 
+
+Committers participate in reviewing code contributions and are able to include them in the project once they have been accepted and verified.
+
+
```

Comment:
Is that a commiter can become a maintainer ? and how can he do that ?

Reply:
Do you mean like a specific process for committers to become maintainer that would be easier than standard process to become maintainer ?

Reply:
@Progi1984 Yes. If this document does not say "committers cannot become maintainers", it means that it's possible.